docs/profiling/numpy_calculator.py

```diff
@@ -0,0 +1,30 @@
+import numpy as np
+from signal_processing_algorithms.e_divisive.base import EDivisiveCalculator
+from signal_processing_algorithms.e_divisive.calculators import numpy_calculator as EDivisive
+
+
+class NumpyCalculator(EDivisiveCalculator):
```

Comment:
Adapted all of these calculators from [here](https://github.com/10gen/signal-processing/pull/50/files#diff-fce2fea32358e2d1313e7a2fe822fdd2L24-L265)